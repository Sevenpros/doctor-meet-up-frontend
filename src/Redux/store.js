import { configureStore } from '@reduxjs/toolkit';
// import logger from 'redux-logger';
import doctorsReducer from './doctors/doctors';
import doctorReducer from './doctors/doctor';

const store = configureStore({
  reducer: {
    doctors: doctorsReducer,
    doctor: doctorReducer,
  },
  // middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});
export default store;
