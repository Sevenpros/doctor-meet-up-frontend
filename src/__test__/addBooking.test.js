import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/store';

import ADDBookings from '../Components/bookings/addBooking';  // eslint-disable-line

describe('ADDBookings tests', () => {
  it('render ADDBookings test', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Router>
          <ADDBookings />
        </Router>
      </Provider>,

    );
    expect(tree).toMatchSnapshot();
  });
});
