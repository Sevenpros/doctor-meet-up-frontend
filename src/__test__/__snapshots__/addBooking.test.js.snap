// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ADDBookings tests render ADDBookings test 1`] = `null`;
